CHANGELOG.md

```diff
@@ -7,14 +7,22 @@
 Cromwell now allows for a user to submit the URL pointing to workflow file to run a workflow using `workflowUrl` parameter. Currently, this is only supported in `Server` mode.
 More details on how to use it can be found [here](http://cromwell.readthedocs.io/en/develop/api/RESTAPI/).
 
+### Languages
+
+- Added an opt-in namespace cache for the WDL Draft 2 language factory. Please see the Cromwell example configuration for details.
```

Comment:
Maybe warn that if people are upgrading from hotfix, their currently enabled cache is now opt-in rather than on-by-default?